forc/src/cli/commands/plugins.rs

```diff
@@ -1,8 +1,38 @@
+use crate::cli::PluginsCommand;
 use anyhow::Result;
+use clap::Parser;
+use std::ffi::OsStr;
+use std::path::PathBuf;
+
+/// Find all forc plugins available via `PATH`.
+///
+/// Prints information about each discovered plugin.
+#[derive(Debug, Parser)]
+pub struct Command {
+    /// Prints the absolute path to each discovered plugin.
+    #[clap(long = "verbose", short = 'v')]
+    print_full_path: bool,
```

Comment:
I think `verbose` is maybe a bit too general/ambiguous for this case? Could we change it to something like `paths`, e.g. `cargo plugins --paths`?